### ISSUE TITLE
predict_trials() returns ground truth labels from Dataset as trial_labels?

### ISSUE THREAD

**Issue by yanlingpan** [braindecode/braindecode#566]
I thought `trial_labels` returned are computed from `trial_predictions`, which should be the same as what returned by calling `predict()`. But it returns the ground truth labels from the dataset. Is this is intentional?

code from `.training.scoring.predict_trials`

```python
loader = DataLoader(
    dataset=dataset,
    batch_size=batch_size,
    shuffle=False,
    num_workers=num_workers,
)
device = next(module.parameters()).device
all_preds, all_ys, all_inds = [], [], []
with torch.no_grad():
    for X, y, ind in loader:
        X = X.to(device)
        preds = module(X)
        all_preds.extend(preds.cpu().numpy().astype(np.float32))
        all_ys.extend(y.cpu().numpy().astype(np.float32))
        all_inds.extend(ind)
```

**Comment by bruAristimunha** [comment]
Hello @yanlingpan,

I am not fully familiar with this function, but I am pinging who implemented them.

**Comment by bruAristimunha** [comment]
Hey @gemeinl and @dengemann!

Can you please help me with this small issue/question about the function predict_trials? 🙏🏽
